export const THROW_ERROR = 'error.THROW'
